README.md

```diff
@@ -21,6 +21,18 @@ gem "twingly-search"
 
 ## Usage
 
+The `twingly-search` gem talks to a commercial API and requires an API key. Best practice is to set the `TWINGLY_SEARCH_KEY` environment variable to the obtained key. `Twingly::Search::Client` and `Twingly::LiveFeed::Client` can be passed a key at initialization if your setup does not allow environment variables.
+
+To learn more about the features of this gem, read the [gem documentation] or check out the example code that can be found in [examples/](examples/).
+
+To learn more about the capabilities of Twingly's APIs, please read the [Search API documentation] and [LiveFeed API documentation].
```

Comment:
`[Search API documentation]` is unspecified, it's `[Twingly Search API Documentation]`. 

Also I think the "official" names are "Twingly Blog Search API" and "Twingly Blog LiveFeed API", but we could shorten it down a bit here, just as long we're consistent.